tools/data-handler/src/interfaces/project-interfaces.ts

```diff
@@ -17,33 +17,33 @@
 export interface dotSchemaContent {
   id: string;
   version: number;
-  cardkeyPrefix?: never;
+  cardKeyPrefix?: never;
   name?: never;
 }
 
-// Project's settings (=cardsconfig.json).
+// Project's settings (=cardsConfig.json).
```

Comment:
Could we capitalize the interfaces? It's fairly common in typescript. https://ts.dev/style/#identifiers

Reply:
Yes, but I will do these separately:
1) change schemas to follow our naming rules
2) change interfaces to follow to follow TypeScript naming rules